src/components/BShuttle/BShuttle.vue

```diff
@@ -0,0 +1,480 @@
+<template>
+  <div class='shuttle'>
+
+    <div class='available'>
+      <label :id="'available-label-' + id">{{ availableLabel }}</label>
+      <ul
+        aria-labeledby='available-label'
+        role='selectbox'
+        tabindex='0'
+      >
+        <li
+          v-if="placeholder && (!options || options.length === 0)"
+          :aria-label="placeholder"
+          class='placeholder'
+        >
+        {{ placeholder }}
+        </li>
+        <li
+          v-for="option in availableOptions"
+          :key="'available-option-' + id + '-' + option.id"
+          :id="'available-option-' + id + '-' + option.id"
+          :class="optionsClass(option, 'available')"
+          @click="onOptionClick(option, 'available', 'chosen', $event)"
+          @keyup.space.prevent.stop="onOptionClick(option, 'available', 'chosen', $event)"
+          @keyup.down.prevent.stop="onKeyupDown($event, option, 'available', 'chosen')"
+          @keyup.up.prevent.stop="onKeyupUp($event, option, 'available', 'chosen')"
+          @keyup.right.prevent.stop="onMoveSelectedClick('available')"
+          @keyup.enter.prevent.stop="onMoveSelectedClick('available')"
+          @keydown.up.prevent.stop
+          @keydown.down.prevent.stop
+          @keydown.space.prevent.stop
+          role='option'
+          tabindex='0'
+        >
+        {{ option.displayText }}
+        </li>
+      </ul>
+    </div>
+
+    <div class='buttons'>
```

Comment:
Can we make these less generic or add `b-` to the front of them? This could cause people to accidentally change styles.

Reply:
thats fair, will do